addons/core/functions/fnc_getSwSettings.sqf

```diff
@@ -65,6 +65,20 @@ if (isNil "_rc") then {
     _value set [TFAR_CODE_OFFSET, _rc];
 };
 
+private _hdo = _value select HALFDUPLEX_OVERRIDE_OFFSET;
```

Comment:
There has to be something for SR Radios. The Range and other stuff is also read from config.